### DISCUSSION TITLE
Last line is not available in full screen mode

### DISCUSSION THREAD

**Discussion by marchank0** [wezterm/wezterm#3520] [Q&A] [answered]
MacOS Monterey 12.6.4, wezterm 20230408-112425-69ae8472:  
`font_size: 16`  
<img width="1439" alt="wez" src="https://user-images.githubusercontent.com/1183579/232033962-eea49ccc-bb89-40c6-8894-23b6b7bfc8f5.png">
`font_size: 15`  
<img width="1439" alt="wez2" src="https://user-images.githubusercontent.com/1183579/232038370-90ba354d-61ea-4c34-9b1e-9980c1f79f73.png">

my config:
```lua
local wezterm = require 'wezterm'
local mux = wezterm.mux

wezterm.on('gui-startup', function(cmd)
    local window = mux.spawn_window(cmd or {})
    window:gui_window():maximize()
end)

return {
    font = wezterm.font('JetBrains Mono'),
    font_size = 15,
    color_scheme = 'dawnfox',
    tab_bar_at_bottom = true,
    hide_tab_bar_if_only_one_tab = true,
    native_macos_fullscreen_mode = true,
    window_padding = { left = 0, right = 0, top = 0, bottom = 0 },
    window_frame = { border_bottom_height = 0 },
    tab_max_width = 22,
    -- window_decorations = "RESIZE" -- hide decoration on mac.
}
```

**Comment by wez** [comment, marked as answer]
the line isn't tall enough to use for a complete line, so it is not used and instead is used as padding to round out to the window size
